Client/mods/deathmatch/logic/CClientPed.cpp

```diff
@@ -6672,6 +6688,40 @@ bool CClientPed::ExitVehicle()
         return false;
     }
 
+    auto targetDoor = g_pGame->GetCarEnterExit()->ComputeTargetDoorToExit(m_pPlayerPed, pOccupiedVehicle->GetGameVehicle());
```

Comment:
Please don't use **auto** for short types like int, bool etc. In this case you can use int8_t (maybe uint8_t because I don't know if -1 is ever returned?) or just int.